src/test/java/com/revaturelabs/ask/AskApplicationServiceTests.java

```diff
@@ -972,36 +1498,47 @@ public void questionsQuestionNotFoundWhenUpdatingTagsExceptionTest() {
     when(questionRepositoryMock.findById(1)).thenReturn(Optional.empty());
     questionServiceImpl.updateTags(testQuestion1);
   }
-  
+
   /**
    * Testing highlighting a response
    */
   @Test
   public void questionsHighlightResponseTest() {
     when(questionRepositoryMock.findById(1)).thenReturn(Optional.of(testQuestion1));
     when(questionRepositoryMock.save(testQuestion1)).thenReturn(testQuestion1);
-    
-    assertEquals((Integer) 4, questionServiceImpl.highlightResponse(1, 4).getHighlightedResponseId());
+
+    assertEquals((Integer) 4,
+        questionServiceImpl.highlightResponse(1, 4).getHighlightedResponseId());
   }
-  
+
   /**
    * Testing failure to find a question when highlighting a response
    */
   @Test(expected = QuestionNotFoundException.class)
   public void questionsQuestionNotFoundWhenHighlightingResponseTest() {
     when(questionRepositoryMock.findById(1)).thenReturn(Optional.empty());
-    
+
     questionServiceImpl.highlightResponse(1, 4);
   }
-  
+
   /**
    * Testing failure to update a response when there is a conflict updating the data
    */
   @Test(expected = QuestionConflictException.class)
   public void questionsConflictExceptionWhenHighlightingResponseTest() {
-    when(questionRepositoryMock.findById(1)).thenReturn(Optional.of(testQuestion1));
-    when(questionRepositoryMock.save(testQuestion1)).thenThrow(DataIntegrityViolationException.class);
-    
+    // when(questionRepositoryMock.findById(1)).thenReturn(Optional.of(testQuestion1));
+    // when(questionServiceImpl.getById(1)).thenReturn((Question)
+    // when(questionRepositoryMock.findById(1)).thenReturn(Optional.of(testQuestion1)));
+    // when(questionServiceImpl.update(testQuestion1)).thenThrow(DataIntegrityViolationException.class);
+    // when(questionRepositoryMock.save(testQuestion1)).thenThrow(DataIntegrityViolationException.class);
```

Comment:
remove

Reply:
Was removed, originally made to showcase testing in the question package.
Ultimately showing how flawed the testing for exceptions occurring within another method is.